iron-iconset-svg.html

```diff
@@ -136,6 +136,10 @@
      */
     _nameChanged: function() {
       new Polymer.IronMeta({type: 'iconset', key: this.name, value: this});
+      this.async(function() {
```

Comment:
Because if the iconset is initializing then there's a bit more processing that it needs to do. If the name was changed after the iconset was already added though then we want to fire this event here.

This seemed like the simplest solution.